elixir_buildpack.config

```diff
@@ -2,7 +2,7 @@
 erlang_version=20.2.1
```

Comment:
We could also upgrade OTP to 21.2.7, could you test it?

Reply:
Sure thing! Let me try that and I'll get back to you.

Reply:
Turns out the buildpack hasn't added [.26](https://github.com/HashNuke/heroku-buildpack-elixir-otp-builds/pull/56) and [.27](https://github.com/HashNuke/heroku-buildpack-elixir-otp-builds/pull/57) yet. Right now that would mean installing 21.2.5 or waiting until those PR's are merged. What would you prefer?

Reply:
We might be a little too strict on versions, any `21.2.*` version could do I guess.

Reply:
Updated with 21.2 as suggested.